e2etests/testcase_test.go

```diff
@@ -3790,7 +3790,7 @@ Applications using RegexRequestMatcher with '.' in the regular expression are po
 				Name:          "openssl",
 				NamespaceName: "ubuntu:22.04",
 				VersionFormat: "dpkg",
-				Version:       "3.0.2-0ubuntu1.6",
```

Comment:
This shouldn't change. This is the version of the package in the image. I think you meant to change "FixedBy"